src/model/ComputationChild.js

```diff
@@ -40,6 +40,8 @@ export default class ComputationChild extends Model {
 			this.dirty = false;
 			const parentValue = this.parent.get();
 			this.value = parentValue ? parentValue[ this.key ] : undefined;
+			if ( this.wrapper ) this.newWrapperValue = this.value;
+			this.adapt();
```

Comment:
As far as I know, adaption happens on set. Not sure how computations work yet. 😕

Reply:
For computations and their children, they get notified any time they change, including on `set`. They then set a dirty flag so that next time a dep asks for the value they know to pull a fresh one. The `Computation` is handling the adapt correctly on`get`, which is when it first knows its actual value, but `ComputationChild` was missing that.

Reply:
I left out a crucial bit: all computations are lazy, so they don't know their value until they are asked for it.

That means that `get` acts kinda like `applyValue` from a regular model too, and `set` calls the setter, invalidates the model, and waits for a dep to ask for the new value before actually updating.

Reply:
> all computations are lazy

Aha! That makes sense now. 😁